### ISSUE TITLE
If you have problem with commiting.

### ISSUE THREAD

**Issue by salmoh** [salmoh/lifelike#17]
You can delete the local clone on your hard drive and then clone the repo again.

**Comment by Noveel** [comment]
I think i have to do so

**Comment by Noveel** [comment]
Have you changed something? I have tried with two different machines

**Comment by salmoh** [comment]
No, I have just updated some files no settings have been changed.

**Comment by salmoh** [comment]
just go to a new directory and clone the repo there. 

    $ git clone git@github.com:salmoh/lifelike.git

**Comment by Noveel** [comment]
ok thanks

**Comment by Noveel** [comment]
now it worked... I couldn't create a directory like before so I used this command:
 git add -f lib/images2gif.py
